files/biothings_studio/bin/hub.py

```diff
@@ -83,4 +83,4 @@
     logging.info("Hub DB backend: %s" % biothings.config.HUB_DB_BACKEND)
     logging.info("Hub database: %s" % biothings.config.DATA_HUB_DB_DATABASE)
 
-    server.start()
```

Comment:
@remoteeng00 don't know why this change (also a few files below). My linter `flake8` will report an error for not having an empty line at the end.